roles/azure_networking/tasks/security_groups.yml

```diff
@@ -122,4 +122,36 @@
   vars:
     security_group_name: "{{ stack_prefix }}_logservers"
   when: security_group_name in security_group_names_of_all_hosts_created_in_azure
+#
+# Repository: receive 8443 from anywhere, block ICMP
+#
```

Comment:
Do we need a new security group for `repo` servers? Previously we had that also for `docs`, `jenkins`, `pulp`, etc., but in the end those are all just web servers. We do not use port 443 as fallback for SSH on those machines: we only do that on the `jumphosts`. So we can use the standard port 443 for HTTPS, which means one security group for web servers is enough. When we use the default port 443 then we also do not need extra logic for the `iptables` firewall with a block for `iptables_allow_repository_inbound` as we can use the existing `iptables_allow_https_inbound`.